src/back/glsl.rs

```diff
@@ -1255,25 +1237,45 @@ fn write_constant(
     })
 }
 
+struct ScalarString<'a> {
+    prefix: &'a str,
```

Comment:
Maybe we should use an `Option` here so that we can support 2 bytes float and others in the future